<div class="flex flex-col h-full -mb-4">
    <div class="flex flex-col items-center h-full mt-6 w-full">
        <div class="flex w-full p-4">
            <a class="font-bold" href="/">
                <h6>
                    <i class="text-xs pi pi-chevron-left"></i>
                    <span class="ml-4 text-[#224063]">Back to Home page</span>
                </h6>
            </a>
        </div>
        <div class="flex w-full p-4 pb-0 grow">
            <nav class="flex w-[200px] flex-col">
                <button
                    class="z-10 flex items-center border-r-2 border-solid p-4 hover:bg-gray-100 active:border-[#495057] text-nowrap"
                    [class.border-[#495057]]="activePanel$.value === ActiveAboutPanel.ABOUT"
                    [class.text-[#224063]]="activePanel$.value === ActiveAboutPanel.ABOUT"
                    (click)="activePanel$.next(ActiveAboutPanel.ABOUT)">
                    <i class="pi pi-info-circle"></i><a class="ml-2 font-bold">About</a>
                </button>
                <button
                    class="z-10 flex items-center border-r-2 border-solid p-4 hover:bg-gray-100 active:border-[#495057] text-nowrap"
                    [class.border-[#495057]]="activePanel$.value === ActiveAboutPanel.TUTORIAL"
                    [class.text-[#224063]]="activePanel$.value === ActiveAboutPanel.TUTORIAL"
                    (click)="activePanel$.next(ActiveAboutPanel.TUTORIAL)">
                    <i class="pi pi-question-circle"></i><a class="ml-2 font-bold">Tutorial</a>
                </button>
                <button
                    class="z-10 flex items-center border-r-2 border-solid p-4 hover:bg-gray-100 active:border-[#495057] text-nowrap"
                    [class.border-[#495057]]="activePanel$.value === ActiveAboutPanel.STATISTICS"
                    [class.text-[#224063]]="activePanel$.value === ActiveAboutPanel.STATISTICS"
                    (click)="activePanel$.next(ActiveAboutPanel.STATISTICS)">
                    <i class="pi pi-chart-bar"></i><a class="ml-2 font-bold">Statistics</a>
                </button>
                <button
                    class="z-10 flex items-center border-r-2 border-solid p-4 hover:bg-gray-100 active:border-[#495057] text-nowrap"
                    [class.border-[#495057]]="activePanel$.value === ActiveAboutPanel.GITHUB"
                    [class.text-[#224063]]="activePanel$.value === ActiveAboutPanel.GITHUB"
                    (click)="activePanel$.next(ActiveAboutPanel.GITHUB)">
                    <i class="pi pi-github"></i><a class="ml-2 font-bold">Github</a>
                </button>
                <button
                    class="z-10 flex items-center border-r-2 border-solid p-4 hover:bg-gray-100 active:border-[#495057] text-nowrap"
                    [class.border-[#495057]]="activePanel$.value === ActiveAboutPanel.PUBLICATIONS"
                    [class.text-[#224063]]="activePanel$.value === ActiveAboutPanel.PUBLICATIONS"
                    (click)="activePanel$.next(ActiveAboutPanel.PUBLICATIONS)">
                    <i class="pi pi-book"></i><a class="ml-2 font-bold">Publications</a>
                </button>
                <button
                    class="z-10 flex items-center border-r-2 border-solid p-4 hover:bg-gray-100 active:border-[#495057] text-nowrap"
                    [class.border-[#495057]]="activePanel$.value === ActiveAboutPanel.TEAM"
                    [class.text-[#224063]]="activePanel$.value === ActiveAboutPanel.TEAM"
                    (click)="activePanel$.next(ActiveAboutPanel.TEAM)">
                    <i class="pi pi-users"></i><a class="ml-2 font-bold">Team</a>
                </button>
                <button
                    class="z-10 flex items-center border-r-2 border-solid p-4 hover:bg-gray-100 active:border-[#495057] text-nowrap"
                    [class.border-[#495057]]="activePanel$.value === ActiveAboutPanel.GET_INVOLVED"
                    [class.text-[#224063]]="activePanel$.value === ActiveAboutPanel.GET_INVOLVED"
                    (click)="activePanel$.next(ActiveAboutPanel.GET_INVOLVED)">
                    <i class="pi pi-comment"></i><a class="ml-2 font-bold">Get Involved</a>
                </button>
            </nav>

            <div class="-ml-0.5 h-full border-l-2 border-solid border-[#e5e7eb]"></div>

            <!-- About -->
            <div>
                <main *ngIf="activePanel$.value === ActiveAboutPanel.ABOUT" class="w-full h-full pb-12 pl-16">
                    <h1 class="mb-4 font-bold">About</h1>
                    <h5 class="leading-xl">
                        Trained on high-quality datasets from UniProt, we developed a machine learning model named CLEAN (Contrastive Learning–Enabled Enzyme Annotation) for enzyme function prediction. CLEAN takes an amino acid sequence as input and outputs a ranked list of predicted enzyme functions represented by Enzyme Commission (EC) numbers. Published in <a class="text-[#0080FF] underline" target="_blank" rel="noopener" href="https://www.science.org/doi/10.1126/science.adf2465">Science (2023)</a>, the CLEAN AI tool combines contrastive learning with pre-trained protein language models to achieve high accuracy predictions, even for enzymes with low sequence similarity to known proteins. CLEAN can confidently (i) annotate understudied enzymes, (ii) correct mislabeled enzymes, and (iii) identify promiscuous enzymes that possess multiple catalytic functions.
                    </h5>

                    <h5 class="leading-xl">
                        Built on top of this powerful tool, the CLEAN Database (CLEAN DB) is an open-access, comprehensive resource containing over 55 million enzyme function predictions and annotations. It offers a user-friendly platform for searching, browsing, and downloading annotations at scale. In addition to function predictions, CLEAN DB enables researchers to assess the effects of mutations using cutting-edge protein language models, making it a powerful tool for both computational and experimental enzyme discovery.
                    </h5>

                    <h5 class="leading-xl">
                        CLEAN DB supports a broad range of scientific applications, including enzyme engineering, synthetic biology, drug discovery, metabolic modeling, and functional genomics. Whether you're identifying candidate biocatalysts or studying the functional impact of mutations, CLEAN DB empowers you to extract meaningful insights from protein sequences at unprecedented scale and accuracy.
                    </h5>
                </main>

                <!-- Tutorial -->
                <main *ngIf="activePanel$.value === ActiveAboutPanel.TUTORIAL" class="w-full h-full pb-12 pl-16">
                    <app-tutorial></app-tutorial>
                </main>

                <!-- Statistics -->
                <main *ngIf="activePanel$.value === ActiveAboutPanel.STATISTICS" class="w-full h-full pb-12 pl-16">
                    <h1 class="mb-4 font-bold">Statistics</h1>
                    <h5 class="leading-xl">
                        The current release of the CLEAN Database provides free access to over 55 million predicted enzyme functions,
                        generated using cutting-edge artificial intelligence tools.
                    </h5>
                    <h5>Learn more about our data below.</h5>

                    <app-data-snapshot></app-data-snapshot>
                </main>

                <!-- Github -->
                <main *ngIf="activePanel$.value === ActiveAboutPanel.GITHUB" class="w-full h-full pb-12 pl-16">
                    <h1 class="mb-4 font-bold">Github</h1>
                    <h5 class="leading-xl">
                        The CLEAN Database (CLEAN DB) is a comprehensive digital repository of predicted enzyme functions, equipped with computational tools for mutation effect prediction. Explore the associated public code repositories below.</h5>

                    <h5><b class="font-bold">Frontend: </b><a rel="noopener" class="text-[#0080FF] underline" target="_blank"
                            href="https://github.com/moleculemaker/cleandb-frontend"><i
                                class="mx-2 pi pi-external-link"></i>https://github.com/moleculemaker/CLEANDB-frontend</a>
                    </h5>
                    <h5><b class="font-bold">CLEAN: </b><a rel="noopener" class="text-[#0080FF] underline" target="_blank"
                            href="https://github.com/tttianhao/CLEAN"><i
                                class="mx-2 pi pi-external-link"></i>https://github.com/tttianhao/CLEAN</a></h5>
                </main>

                <!-- Publications -->
                <main *ngIf="activePanel$.value === ActiveAboutPanel.PUBLICATIONS" class="w-full h-full pb-12 pl-16">
                    <h1 class="mb-4 font-bold">Publications</h1>
                    <h5 class="leading-xl">
                        Learn more about the in-house CLEAN AI tool that powered the creation of the CLEAN Database.
                    </h5>

                    <ul class="mt-8">
                        <li class="items-center mt-4">
                            <h5 class="mt-0 leading-xl"><b class="font-semibold">CLEAN AI:</b> Tianhao Yu et al., Enzyme function prediction using contrastive learning. Science 379, 1358-1363 (2023).</h5>
                            <h5 class="mt-0 leading-xl">
                                <a rel="noopener" class="text-[#0080FF] underline" target="_blank"
                                    href="https://doi.org/10.1126/science.adf2465"><i
                                        class="mx-2 pi pi-external-link"></i>DOI:https://doi.org/10.1126/science.adf2465</a>
                            </h5>
                        </li>
                    </ul>
                </main>

                <!-- Team -->
                <main *ngIf="activePanel$.value === ActiveAboutPanel.TEAM" class="w-full h-full pb-12 pl-16">
                    <h1 class="mb-4 font-bold">Team</h1>
                    <h5 class="leading-xl">
                        Meet the team behind CLEAN Database.
                    </h5>

                    <div class="flex mt-8">
                        <img src="assets/team1.png" class="w-[96px] h-[96px] bg-white rounded-full" alt="Huimin Zhao" />
                        <div class="ml-8">
                            <h5 class="mt-0 font-bold leading-xl">Huimin Zhao</h5>
                            <h6 class="mt-1 leading-xl font-semibold text-[#6C757D]">Project Director</h6>
                            <h6 class="mt-2 leading-xl">
                                Dr. Huimin Zhao <a rel="noopener" class="text-[#0080FF]" href="https://chbe.illinois.edu/people/profile/zhao5"
                                target="_blank">(https://chbe.illinois.edu/people/profile/zhao5)</a> is the Steven L. Miller Chair Professor at the University of Illinois at Urbana-Champaign (UIUC). He is currently the director of NSF AI Research Institute for Molecule Synthesis <a rel="noopener" class="text-[#0080FF]"
                                href="https://moleculemaker.org" target="_blank">(https://moleculemaker.org)</a>, the director of NSF iBioFoundry, and the director of NSF Global Center for Reliable and Scalable Biofoundries. He received his Ph.D. degree in Chemistry from the California Institute of Technology in 1998 under the guidance of Nobel Laureate Dr. Frances Arnold. Dr. Zhao has authored and co-authored over 453 research articles and over 30 issued and pending patent applications with several being licensed by industry. Dr. Zhao is the Editor-in-Chief of ACS Synthetic Biology and the Founding Associate Editor of ACS Catalysis (2011-2022). His primary research interests are in the development and applications of synthetic biology, machine learning, and laboratory automation tools to address society’s most daunting challenges in health, energy, and sustainability.
                            </h6>
                        </div>
                    </div>

                    <div class="flex mt-8">
                        <img src="assets/team2.png" class="w-[96px] h-[96px] object-cover rounded-full bg-white" alt="Le Yuan" />
                        <div class="ml-8">
                            <h5 class="mt-0 font-bold leading-xl">Le Yuan</h5>
                            <h6 class="mt-1 leading-xl font-semibold text-[#6C757D]">Postdoc UIUC</h6>
                            <h6 class="mt-2 leading-xl">Dr. Le Yuan is a postdoctoral researcher at the Carl R. Woese
                                Institute of Genomic Biology (IGB) at the University of Illinois at Urbana-Champaign,
                                under
                                the supervision of Prof. Huimin Zhao. Prior to this, he completed his Ph.D. degree in
                                the
                                Department of Life Sciences at Chalmers University of Technology. Dr. Yuan has published
                                16
                                academic papers in prestigious journals, including Nature Catalysis, Nature Protocols,
                                Nature Communications, etc. His primary research interests include synthetic biology,
                                systems biology and machine learning.</h6>
                        </div>
                    </div>

                    <div class="flex mt-8">
                        <img src="assets/david.jpg" class="w-[96px] h-[96px] object-cover rounded-full bg-white" alt="David Bianchi"/>
                        <div class="ml-8">
                            <h5 class="mt-0 font-bold leading-xl">David Bianchi</h5>
                            <h6 class="mt-1 leading-xl font-semibold text-[#6C757D]">Research Scientist</h6>
                            <h6 class="mt-2 leading-xl">Dr. David Bianchi is a research scientist at the National Center for Supercomputing Applications (NCSA) at the University of Illinois Urbana-Champaign. Prior to this, he completed his Ph.D. degree in the Department of Chemistry at UIUC. His work focuses on development of computational science platforms, analyses and research with applications ranging from chemistry to genomics, agriculture, health sciences, energy, synthetic biology and bioengineering, and applications of machine learning to these areas.</h6>
                        </div>
                    </div>

                    <div class="flex mt-8">
                        <img src="assets/MattBerry1.jpg" class="w-[96px] h-[96px] object-cover rounded-full bg-white" alt="Matt Berry"/>
                        <div class="ml-8">
                            <h5 class="mt-0 font-bold leading-xl">Matt Berry</h5>
                            <h6 class="mt-1 leading-xl font-semibold text-[#6C757D]">Lead Software Engineer</h6>
                            <h6 class="mt-2 leading-xl">Matt Berry is the head of Visual Analytics at the National Center for Supercomputing Applications and has served as Lead Research Software Engineer on many major grant- and industry-funded initiatives. His primary interest is the creation of user-friendly, production-grade software systems to help knowledge workers understand and act upon complex data.</h6>
                        </div>
                    </div>

                    <div class="flex mt-8">
                        <img src="assets/bingji.jpg" class="w-[96px] h-[96px] object-cover rounded-full bg-white" alt="Bingji Guo"/>
                        <div class="ml-8">
                            <h5 class="mt-0 font-bold leading-xl">Bingji Guo</h5>
                            <h6 class="mt-1 leading-xl font-semibold text-[#6C757D]">Research Software Engineer</h6>
                            <h6 class="mt-2 leading-xl">Bingji Guo is a Research Software Engineer at the National Center for Supercomputing Applications (NCSA) at the University of Illinois Urbana-Champaign. His work focuses on scientific data visualization, frontend development, and integrating research workflows with effective software technologies. He contributes to many tools within the AlphaSynthesis project, enhancing usability and performance for scientific applications.</h6>
                        </div>
                    </div>

                    <div class="flex mt-8">
                        <img src="assets/Sara.png" class="w-[96px] h-[96px] object-cover rounded-full bg-white" alt="Sara Lambert"/>
                        <div class="ml-8">
                            <h5 class="mt-0 font-bold leading-xl">Sara Lambert</h5>
                            <h6 class="mt-1 leading-xl font-semibold text-[#6C757D]">Research Programmer</h6>
                            <h6 class="mt-2 leading-xl">Sara is a Research Programmer at the National Center for Supercomputing Applications who has contributed to many of the tools within the AlphaSynthesis project, enhancing usability and performance of scientific applications.</h6>
                        </div>
                    </div>

                    <div class="flex mt-8">
                        <img src="assets/chris.jpg" class="w-[96px] h-[96px] object-cover rounded-full bg-white" alt="Chris Pond"/>
                        <div class="ml-8">
                            <h5 class="mt-0 font-bold leading-xl">Chris Pond</h5>
                            <h6 class="mt-1 leading-xl font-semibold text-[#6C757D]">Engineering Manager</h6>
                            <h6 class="mt-2 leading-xl">Chris Pond is an Engineering Manager and Lead Data Engineer at the National Center for Supercomputing Applications (NCSA) at the University of Illinois Urbana-Champaign. His work spans project leadership, data engineering, and software development across healthcare, AI, and high-performance computing. He has contributed to major initiatives including Technical Integrator for ARPA-H MarginDx, Technical PI for the Mayo Clinic CORE project, the Dark Energy Survey, and the Vera C. Rubin Observatory. He also led the creation and development of the NCSA COVID LIMS, a custom laboratory information management system supporting the University of Illinois COVID-19 saliva testing program.</h6>
                        </div>
                    </div>

                    <div class="flex mt-8">
                        <img src="assets/stephens.jpeg" class="w-[96px] h-[96px] object-cover rounded-full bg-white" alt="Chris Stephens"/>
                        <div class="ml-8">
                            <h5 class="mt-0 font-bold leading-xl">Chris Stephens</h5>
                            <h6 class="mt-1 leading-xl font-semibold text-[#6C757D]">Research Data Engineer</h6>
                            <h6 class="mt-2 leading-xl">Chris Stephens is a Lead Research Data Engineer at NCSA (University of Illinois Urbana-Champaign) with over 25 years of experience in data-centric projects. He designs and implements data pipelines supporting scientific and medical research across disciplines.
                                He has contributed to major initiatives including ARPA-H MarginDx, Mayo Clinic CORE, the Dark Energy Survey, and the Vera C. Rubin Observatory. During the pandemic, Chris helped develop the NCSA COVID LIMS that supported the university's COVID-19 testing program.
                                Chris effectively transforms complex research requirements into efficient data systems through technical expertise and collaborative problem-solving.</h6>
                        </div>
                    </div>

                    <div class="flex mt-8">
                        <img src="assets/Kate.jpeg" class="w-[96px] h-[96px] object-cover rounded-full bg-white" alt="Kate Arneson"/>
                        <div class="ml-8">
                            <h5 class="mt-0 font-bold leading-xl">Kate Arneson</h5>
                            <h6 class="mt-1 leading-xl font-semibold text-[#6C757D]">Research UIX Designer</h6>
                            <h6 class="mt-2 leading-xl">Kate Arneson is a research user interface and experience (UIX) designer at the National Center for Supercomputing Applications (NCSA) at the University of Illinois Urbana-Champaign. Her work focuses on applying human-centered design to research software projects, with applications ranging from chemistry and machine learning to genomics and healthcare. At NCSA and the Molecule Maker Lab Institute, Kate collaborates closely with scientists, engineers, and end-users to create usable, reproducible, and sustainable applications, advancing the role of design in interdisciplinary research.</h6>
                        </div>
                    </div>

                    <div class="flex mt-8">
                        <img src="assets/Chenxin.jpg" class="w-[96px] h-[96px] object-cover rounded-full bg-white" alt="Chenxin Wang"/>
                        <div class="ml-8">
                            <h5 class="mt-0 font-bold leading-xl">Chenxin Wang</h5>
                            <h6 class="mt-1 leading-xl font-semibold text-[#6C757D]">Academic Hourly</h6>
                            <h6 class="mt-2 leading-xl">Chenxin Wang is an Academic Hourly at the National Center for Supercomputing Applications (NCSA) at the University of Illinois Urbana-Champaign. His work focuses on developing web-based tools that support and streamline scientific research workflows. His contributions help bridge the gap between complex scientific processes and accessible digital platforms, enhancing research productivity and user experience.</h6>
                        </div>
                    </div>

                    <div class="flex mt-8s">
                        <img src="assets/LeighFu.jpg" class="w-[96px] h-[96px] object-cover rounded-full bg-white" alt="Leigh Fu"/>
                        <div class="ml-8">
                            <h5 class="mt-0 font-bold leading-xl">Leigh Fu</h5>
                            <h6 class="mt-1 leading-xl font-semibold text-[#6C757D]">Research UIX Designer</h6>
                            <h6 class="mt-2 leading-xl">Leigh Fu is a research user interface and experience (UIX) designer at the National Center for Supercomputing Applications (NCSA) at the University of Illinois Urbana-Champaign.  Her work focuses on translating complex scientific workflows into accessible and intuitive interfaces. She works closely with interdisciplinary teams to improve usability, reproducibility, and accessibility—and to ensure that thoughtful design plays a meaningful role in the advancement of scientific software.</h6>
                        </div>
                    </div>
                </main>

                <!-- Get Involved -->
                <main *ngIf="activePanel$.value === ActiveAboutPanel.GET_INVOLVED" class="w-full h-full pb-12 pl-16">
                    <h1 class="mb-4 font-bold">Get involved</h1>
                    <h5 class="leading-xl">
                        We welcome your questions, suggestions, and feedback! Get in touch with us via email:
                    </h5>
                    <h5 class="mt-0 leading-xl">
                        <a class="text-[#0080FF] underline"
                            href="mailto:cleandb-feedback@moleculemaker.org">cleandb-feedback&#64;moleculemaker.org</a>
                    </h5>
                </main>
            </div>

        </div>
    </div>
</div>