<div class="mb-10">
    <h1 class="mb-4 font-bold">Tutorial</h1>
    <h5 class="leading-xl">
        The CLEAN Database (CLEAN DB) provides comprehensive enzyme function and sequence data, along with advanced models for predicting mutation effects, in a unified web-based interface. Below, you’ll find descriptions and examples showing how to use CLEAN DB effectively.
    </h5>

    <h5 class="mt-10 font-semibold">Database Search</h5>
    <h5 class="leading-xl">
        The database search functionality allows users to explore over 55 million predicted enzyme functions. CLEAN DB can support searches by UniProt accession, protein name, gene name, organism, or EC number, directing users to detailed results pages featuring enzyme function predictions and annotations. On the results page, users can filter and sort detailed entries, and explore the types of enzymatic reactions that can be catalyzed. Cross-references to external databases such as UniProt, as well as bulk-download options, are also available.
    </h5>

    <img src="assets/screenshots/cleandb-dbsearch-inputpage-protname.png" class="mt-8 w-[100%]" />
    <h6 class="text-[#6C757D] italic text-center">Multiple input types (UniProt accession, protein name, gene name, organism, or EC number) are supported for database search. Here, the ...[example]... has been input.</h6>

    <img src="assets/screenshots/cleandb-dbsearch-resultspage-protname.png" class="mt-8 w-[100%]" alt="Results page showing enzyme function predictions in a table, with expandable rows for additional information such as reference data and reaction schemes." />
    <h6 class="text-[#6C757D] italic text-center">After clicking ‘Search CLEAN DB,’ relevant results are displayed in an ‘Enzyme Function Prediction’ table. Users can expand individual table rows to access additional information such as reference data and reaction schemes.</h6>

    <h5 class="mt-10 font-semibold">Mutation Effect Prediction</h5>
    <h5 class="leading-xl">
        The mutation effect prediction functionality enables users to generate mutation effect predictions using state-of-the-art protein language models (i.e., ESM-2). After inputting an enzyme sequence, users receive residue-specific prediction results which can be explored through interactive heatmap and tabular data visualizations.
    </h5>

    <img src="assets/screenshots/cleandb-MEP-input.png" class="mt-8 w-[100%]" />
    <h6 class="text-[#6C757D] italic text-center">The analysis of enzyme sequences and amino acid position(s) is supported. Here, the enzyme sequence ...[example]... (FASTA) and positions ...[example]... have been input for mutation effect prediction.</h6>

    <img src="assets/screenshots/cleandb-MEP-results-wRangeSelection.png" class="mt-8 w-[100%]" alt="Prediction results for the specified enzyme sequence and positions, displayed as interactive heatmap and tabular data visualizations." />
    <h6 class="text-[#6C757D] italic text-center">After a short processing time, prediction results are returned for the specified enzyme sequence and/or position(s). Users can explore the predicted mutation effects through interactive heatmap and tabular data visualizations.</h6>
</div>